tests/projects/local-kit/variants/hello-ootb/Cargo.toml

```diff
@@ -15,12 +15,14 @@ xfs-data-partition = true
 systemd-networkd = true
 
 [package.metadata.build-variant]
-included-packages = ["hello-agent", "hello-go"]
+included-packages = ["pkg-a", "pkg-b", "pkg-c", "pkg-d"]
 included-kits = ["hello-kit"]
 kernel-parameters = []
 
 [lib]
 path = "../variants.rs"
 
 [build-dependencies]
-hello-kit = { path = "../../kits/hello-kit" }
+core-kit = { path = "../../kits/core-kit" }
+extra-1-kit = { path = "../../kits/extra-1-kit" }
+extra-2-kit = { path = "../../kits/extra-2-kit" }
```

Comment:
This should work with only one kit dependency right? Like if we add `extra-3-kit` that depends on one and two, and implicitly depends on core, then they should all be available for variant builds.

Reply:
Yes. I think it would be weird, but supported, to include a package from core-kit in the variant's list of packages, then rely on core-kit being a transient dependency of another kit. But it would work.